examples/erc721/src/lib.rs

```diff
@@ -151,4 +151,9 @@ impl Erc721Example {
 
         Ok(())
     }
+
+    pub fn supports_interface(interface_id: FixedBytes<4>) -> bool {
+        Erc721::supports_interface(interface_id)
+            || Enumerable::supports_interface(interface_id)
```

Comment:
Why `Enumerable`?

Reply:
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/4764ea50750d8bda9096e833706beba86918b163/contracts/token/ERC721/extensions/ERC721Enumerable.sol#L39

Reply:
Basically it is [implemented](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/530179a71f435e85ae9df9b9f12b5637cf229e5c/contracts/token/ERC721/extensions/ERC721Enumerable.sol#L39) for Erc721Enumerable in solidity